### DISCUSSION TITLE
Improved request-response logging

### DISCUSSION THREAD

**Discussion by kdgrant** [Dynatrace/dynatrace-configuration-as-code#477] [Ideas]
I'm finding the request/response logging facility a bit clunky to use, since the request & response end up in different log files and need manual handling to correlate: 

```
MONACO_REQUEST_LOG=./request-log.txt MONACO_RESPONSE_LOG=./response-log.txt ... 
```

When specifying the same file for both requests and responses (```MONACO_REQUEST_LOG=./log.txt MONACO_RESPONSE_LOG=./log.txt```) the output is arbitrarily interleaved and truncated, so this isn't a solution (it was worth a try!).

**Suggestion**
Support a ```MONACO_REQUEST_RESPONSE_LOG``` environment variable.

When this is present, requests and responses would be written to the named file as request-response pairs, ordered chronologically by request time.

**Comment by didiladi** [comment]
@kdgrant That's a good idea!
